app/views/admin/users/index.html.erb

```diff
@@ -24,12 +24,13 @@
               <td><%= link_to user.username, edit_admin_user_path(user) %></td>
               <td><%= user.email %></td>
               <td><%= user_account_state(user) %></td>
-              <td><%= user.agents.active.count %></td>
+              <td><%= link_to user.agents.active.count, switch_to_user_admin_user_path(user), data: { confirm: 'This will log you in as another user. Would you like to continue?' } %></td>
```

Comment:
It redirects to the users `agents_path` which will show all agents but what mainly bothers me is that two links do the same thing and I find the "Become User" button more intuitive then the link on the active agent count.